{"status":"FAIL","msg":"FAIL:\n Uncaught: { message: \"Expected a Test262Error but got a ReferenceError (Testing with Function.)\" }\n    at ../../test262/harness/testBigIntTypedArray.js:37\n    at testWithBigIntTypedArrayConstructors (../../test262/test/built-ins/TypedArrayConstructors/ctors-bigint/object-arg/throws-setting-property.js:34)\n\n","path":"../../test262/test/built-ins/TypedArrayConstructors/ctors-bigint/object-arg/throws-setting-property.js","memory_kb":0,"duration":159645415}